galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -430,7 +430,7 @@ private void generateEnvironment(TestClassWrapper testClassWrapper, TestRunManag
         }
 
         try {
-            updateStatus("generating", null);
+            updateStatus(TestRunLifecycleStatus.GENERATING, null);
```

Comment:
Why isn't this following the same pattern of "generating" instead of the null ?

Reply:
Been over this in person, however a comment to help jog memory: the second parameter here is passed in like a boolean, if it's not null then the timestamp for changing of states is saved with the progress of the test, the string being the progress (i.e built means environment has been set up for the test or something). Again we will try to change this in another issue.